app/src/main/java/com/github/polypoly/app/network/FakeRemoteStorage.kt

```diff
@@ -22,28 +82,65 @@ class FakeRemoteStorage : IRemoteStorage {
         stats = Stats(0, 0, 67)
     )
 
-    @RequiresApi(Build.VERSION_CODES.O)
+    init {
+        gameLobbyFull.addUser(testUser2)
+        gameLobbyFull.addUser(testUser3)
+        gameLobbyFull.addUser(testUser4)
+        gameLobbyFull.addUser(testUser5)
+
+        gameLobbyJoinable1.addUser(testUser2)
+        gameLobbyJoinable1.addUser(testUser3)
+
+        gameLobbyJoinable2.addUser(testUser2)
+        gameLobbyJoinable2.addUser(testUser3)
+        gameLobbyJoinable2.addUser(testUser4)
+
+        gameLobbyPrivate.addUser(testUser2)
+        gameLobbyPrivate.addUser(testUser3)
+        gameLobbyPrivate.addUser(testUser4)
```

Comment:
This could be refactored with a function `addUsersToGroups` to make one-line assignments instead per `GameLobby` instead.